server/src/main/java/org/schlunzis/kurtama/server/lobby/LobbyService.java

```diff
@@ -47,7 +48,12 @@ public void onJoinLobbyRequest(ClientMessageWrapper<JoinLobbyRequest> cmw) {
         try {
             ServerLobby lobby = lobbyManagement.joinLobby(request.lobbyID(), cmw.user());
             sendTo(new JoinLobbySuccessfullyResponse(lobby.toDTO()), cmw.user());
-            // TODO update clients of users in lobby #8
+
+            // inform all other users in lobby
+            Collection<ServerUser> users = lobby.getUsers();
+            users.remove(cmw.user());
+            sendToMany(new UserJoinedLobbyMessage(cmw.user().toDTO()), users);
+
```

Comment:
Hier können wir eine `sendToAllInLobby(UUID, IServerMessage)` Methode einführen.

Reply:
Das hier ist ja nicht eine reine `sendToAllInLobby`-Aktion. Hier wird ja die Person, die der Lobby beitritt oder austritt nicht mit der Nachricht angepingt. Das habe ich gemacht, da diese Person schon vorher mit einer Nachricht über die erfolgreiche Aktion informiert wird. Dadurch sparen wir uns eine Nachricht an diese Person. Das Thema haben wir ja schon in #81 tangiert. Wir würden hier doppelte Informationen und mehrere Nachrichten an einen User schicken, die man in einer Nachricht zusammenfassen oder sogar ganz weglassen kann, da sie aus einer anderen Nachricht erschlossen werden können.

Reply:
Das hier ist ja nicht eine reine `sendToAllInLobby`-Aktion. Hier wird ja die Person, die der Lobby beitritt oder austritt nicht mit der Nachricht angepingt. Das habe ich gemacht, da diese Person schon vorher mit einer Nachricht über die erfolgreiche Aktion informiert wird. Dadurch sparen wir uns eine Nachricht an diese Person. Das Thema haben wir ja schon in #81 tangiert. Wir würden hier doppelte Informationen und mehrere Nachrichten an einen User schicken, die man in einer Nachricht zusammenfassen oder sogar ganz weglassen kann, da sie aus einer anderen Nachricht erschlossen werden können.